docs/dataset_fields_for_iris_timestamp.md

```diff
@@ -0,0 +1,112 @@
+# Dataset Fields for IRIS Timestamp
+
+This document maps timestamp field names to their corresponding datasets.
+
+All datasets are case sensitive.
+
+## acceptanceTime
+- **BOALF** -- Bid Offer Acceptance Level Flagged
+
+## createdDateTime
+- **BOAV** -- Bid Offer Acceptance Level Flagged
+- **DISEBSP** -- System Prices
+- **DISPTAV** -- Indicative Volumes
+- **EBOCF** -- Indicative Cashflows
+- **ISPSTACK** -- Bid/Offer Stack
+- **SMSG** -- System Messages
+
+## createdTime
+- **REMIT** -- Regulation on wholesale Energy Markets Integrity and Transparency
+
+## measurementTime
+- **FREQ** -- System Frequency Data
+
+## publishTime
+- **ABUC** -- Amount Of Balancing Reserves Under Contract
+- **AGPT** -- Actual Aggregated Generation Per Type
+- **AGWS** -- Actual or Estimated Wind and Solar Generation
+- **AOBE** -- Accepted Offered Balancing Energy
+- **ATL** -- Actual Total Load Per Bidding Zone
+- **BEB** -- Balancing Energy Bids
+- **CBS** -- Current Balancing State
+- **CCM** -- Costs Of Congestion Managemen
+- **DAG** -- Day-Ahead Aggregated Generation
+- **DATL** -- Day-Ahead Total Load Forecast Per Bidding Zone
+- **DCI** -- Demand Control Instructions
+- **DGWS** -- Day-Ahead Generation Forecasts for Wind and Solar
+- **E0361** -- Credit Default Notices
+- **FEIB** -- Financial Expenses and Income For Balancing
+- **FOU2T14D** -- National output forecast by fuel type (2- 14 days)
+- **FOU2T3YW** -- National output forecast by fuel type (2 - 156 weeks)
+- **FUELHH** -- Half Hour Generation by Fuel Type
+- **FUELINST** -- Instantaneous Generation by Fuel Type
+- **IGCA** -- Installed Generation Capacity Aggregated
+- **IGCPU** -- Installed Generation Capacity Per Unit
+- **IMBALNGC** -- Day and Day-Ahead Imbalance
+- **INDDEM** -- Day and Day-Ahead Indicated Demand
+- **INDGEN** -- Day and Day-Ahead Indicated Generation
+- **INDO** -- Initial Demand Outturn
+- **INDOD** -- Daily Energy Transimitted/ Initial National Demand Outturn Daily
+- **ITSDO** -- Initial Transmission System Demand Outturn
+- **LOLPDM** -- Loss of Load Probability (LoLP) and De-rated Margin
+- **MATL** -- Month-Ahead Total Load Forecast Per Bidding Zone
+- **MELNGC** -- 	Day and Day-Ahead Margin
```

Comment:
I don't really understand why there's whitespace in this line, in the preview it looks fine, but might be worth checking! :)